apps/meteor/client/views/room/hooks/useE2EEResetRoomKey.ts

```diff
@@ -0,0 +1,36 @@
+import type { RoomID } from '@rocket.chat/core-typings';
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import type { UseMutationOptions, UseMutationResult } from '@tanstack/react-query';
+import { useMutation } from '@tanstack/react-query';
+
+import { e2e } from '../../../../app/e2e/client';
+import { dispatchToastMessage } from '../../../lib/toast';
+
+type UseE2EEResetRoomKeyRoomVariables = {
+	roomId: RoomID;
+};
+
+export const useE2EEResetRoomKeyRoom = (
```

Comment:
Shouldn't this hook be called `useE2EEResetRoomKey`?

Reply:
Yep, it should! haha thanks for pointing that